src/directives/button.ts

```diff
@@ -0,0 +1,33 @@
+import { DirectiveOptions } from 'vue'
+
+import KeyValues from '../utils/KeyValues'
+
+const onEnterKeyPress = (e: any) => {
+  if (KeyValues.isEnter(e.key)) {
+    e.preventDefault()
+    e.target.click()
+  }
+}
+
+export const Button = {
+  bind (el, binding, vnode) {
+    if (binding.value || typeof binding.value === 'undefined') {
+      const button = el as HTMLDivElement
+      button.setAttribute('role', 'button')
+      button.addEventListener(
+        'keypress',
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event
MDN recommends to use `keydown  `instead of `keypress ` (deprecated)